doc/lua_api.txt

```diff
@@ -8751,8 +8755,258 @@ Used by `minetest.add_particlespawner`.
+    the position of an entity rather than a coordinate in world space.
+  * float range `strength`: the speed with which particles will move towards
+    `attractor`. If negative, the particles will instead move away from that
+    point.
+  * bool `die_on_contact`: if true, the particles' lifetimes are adjusted so
+    that they will die as they cross the attractor threshold. this behavior
+    is the default but is undesirable for some kinds of animations; set it to
+    false to allow particles to live out their natural lives.
+* vec3 range `radius`: if set, particles will be arranged in a sphere around
+  `pos`. A constant can be used to create a spherical shell of particles, a
+  vector to create an ovoid shell, and a range to create a volume; e.g.
+  `{min = 0.5, max = 1, bias = 1}` will allow particles to appear between 0.5
+  and 1 nodes away from `pos` but will cluster them towards the center of the
+  sphere. Usually if `radius` is used, `pos` should be a single point, but it
+  can still be a range if you really know what you're doing (e.g. to create a
+  "roundcube" emitter volume).
+
+### Textures
+In versions before v5.5.0, particlespawner textures could only be specified as a single
+texture string. After v5.5.0, textures can now be specified as a table as well. This
+table contains options that allow simple animations to be applied to the texture.
+
+    texture = {
+        img = 'mymod_particle_texture.png';
+        -- the texture specification string
+
+        alpha = 1.0;
+        -- controls how visible the particle is; at 1.0 the particle is fully
+        -- visible, at 0, it is completely invisible.
+
+        scale = {x = 0, y = 0};
+        -- scales the texture onscreen
+
+        fade_start = 0.0;
+        -- controls where along the particle's lifetime the fading effect begins,
+        -- 0.0 being the beginning and 1.0 being the end
+
+        fade_reps = 1;
+        -- controls how many times the alpha animation will be repeated over the
+        -- particle's lifetime; allowing e.g. a particle to be set to pulse or
+        -- flicker 5 times before disappearing for good.
+
+        animation = {Tile Animation definition};
+        -- overrides the particlespawner's global animation property for a single
+        -- specific texture
+    }
+
+Instead of setting a single texture definition, it is also possible to set a
+`texpool` property. A `texpool` consists of a list of possible particle textures.
+Every time a particle is spawned, the engine will pick a texture at random from
+the `texpool` and assign it as that particle's texture. You can also specify a
+`texture` in addition to a `texpool`; the `texture` value will be ignored on newer
+clients but will be sent to older (pre-v5.5.0) clients that do not implement
+texpools.
+
+    texpool = {
+        'mymod_particle_texture.png';
+        { img = 'mymod_spark.png', fade = 'out' };
```

Comment:
seems to be. i remember little about this but i think this was a primitive mechanism i replaced by adding more precise controls to the tween tables, `{start, style, reps}`. in current syntax this would be expressed:

```
texpool = {
    {name='mymod_spark.png', alpha_tween={1,0, style='fwd'}} -- 'fwd' is the default
}
```